packages/webapp/pages/squads/[handle]/create.tsx

```diff
@@ -39,7 +39,7 @@ function CreatePost(): ReactElement {
       onAskConfirmation(false);
     },
     onSuccess: async (post) => {
-      const key = generateWritePostKey();
+      const key = generateWritePostKey(squad?.id);
```

Comment:
I suggest we actually return a `clearDraft` on the `useDiscardPost` and pass it down lang the `updateDraft`, this is to avoid ending up in this same situation in the future.

Reply:
Good idea